docs/learn/test-runner/code-coverage/index.md

```diff
@@ -0,0 +1,221 @@
+---
```

Comment:
this is the same?

Reply:
yes the first part of the learn section is in most cases very simliar in the blog as well